test/cartesian/Area.spec.tsx

```diff
@@ -58,7 +58,7 @@ describe.each(chartsThatSupportArea)('<Area /> as a child of $testName', ({ Char
       };
 
       render(
-        <ChartElement width={500} height={500} data={data}>
```

Comment:
Why are we getting rid of height and width? I'm missing something

Reply:
I moved these to the test case definition because I didn't want to repeat it again.

Reply:
Ah thanks